connector/servicegraphconnector/connector_test.go

```diff
@@ -143,16 +143,16 @@ func verifyCount(t *testing.T, m pmetric.Metric) {
 	verifyAttr(t, attributes, "client_some-attribute", "val")
 }
 
-func verifyDuration(t *testing.T, m pmetric.Metric, durationSum float64) {
+func verifyDuration(t *testing.T, m pmetric.Metric, durationSum float64, bucktes []uint64) {
```

Comment:
```suggestion
func verifyDuration(t *testing.T, m pmetric.Metric, durationSum float64, bs []uint64) {
```

Reply:
Even if the suggestion isn't accepted, I would insist on not introducing a typo here (`bucktes`) :-)